src/coreclr/debug/ee/controller.cpp

```diff
@@ -6039,6 +6056,11 @@ bool DebuggerStepper::TrapStep(ControllerStackInfo *info, bool in)
                     fCallingIntoFunclet = IsAddrWithinMethodIncludingFunclet(ji, info->m_activeFrame.md, walker.GetNextIP()) &&
                         ((CORDB_ADDRESS)(SIZE_T)walker.GetNextIP() != ji->m_addrOfCode);
 #endif
+                    if (in && IsStoreTailCallArgs(walker.GetNextIP()))
```

Comment:
Nit: I am just wondering if you want to add some kind of comment regarding which case we are in. This function seems to have a comment about each case.